src/main/webapp/view/transcript/transcript.js

```diff
@@ -30,6 +30,9 @@ export function deleteTranscript() {
 export class TranscriptLineElement extends HTMLElement {
   static #DEFAULT_FONT_WEIGHT = 'text-muted';
   static #BOLD_FONT_WEIGHT = 'font-weight-bold';
+  static #TIME_OFFSET = 1;
```

Comment:
Can you add a comment explaining what this is for?  And include units if this is in seconds/milliseconds/etc.

Reply:
done, thanks!